index.html

```diff
@@ -534,17 +532,21 @@ <h2>Issuer</h2>
         </dl>
         <pre class="example nohighlight" title="Usage of issuer properties">
 {
-  "id": "http://dmv.example.gov/credentials/3732",
-  "type": ["Credential", "ProofOfAgeCredential"],
-  <span class="highlight">"issuer": "https://dmv.example.gov/issuers/14"</span>,
-  <span class="highlight">"issued": "2010-01-01T19:73:24Z"</span>,
-  "claim": {
-    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-    "ageOver": 21
-  },
-  "proof": { ... }
-}
-        </pre>
+"verifiableCredential": {
+	"id": "< credential id >",
+	"type": ["ProofOfAgeCredential"],
```

Comment:
If we remove "Credential", then generalized processing libraries won't be able to key off of that to process the data structure.

Reply:
But they can key off the JSON type of verifiableCredential or verifiableProfile. This also means that we have tight control over which top level types are standardised. With the current data structure anyone can define their own type (e.g. unverifiableRevocationList) and this would be perfectly acceptable and in accordance to the standard. Surely you are not advocating this?

Reply:
@dlongley I think you are confusing two things: the ontology and vocabulary for properties of claims which will be defined by communities of users; and the top level data structures that are being standardised. So I agree that communities will define their own vocabularies of claim properties, but they should not define their own top level data structures. Currently we have just two, VCs and VPs, and these should be fixed until we see a requirement for some other type of data structure. Then that can be standardised as well. The current data model does not place any restrictions on the top level data types/structures. This loophole should be closed.

Reply:
>  The current data model does not place any restrictions on the top level data types/structures. This loophole should be closed.

This is the core of the miscommunication and the disconnect. You are advocating something that no other W3C Linked Data specification has done. This is a discussion that has happened time and time again over the past 15 years at W3C and unfortunately, I think we're going to have to repeat that discussion to come to consensus on this item. @David-Chadwick, please raise an issue for this and we can move the discussion there.